Cargo.toml

```diff
@@ -18,7 +18,8 @@ members = [
 ]
 
 [package.metadata.docs.rs]
-features = [ "qr", "serde_support", "gen_secret" ]
+all-features = true
+rustc-args = ["--cfg", "docsrs"]
```

Comment:
I'll admit I've never went too far with rustdocs. This is nice to know, thank you :)